.jenkins/pipelines/Azure/Linux/Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
-library "OpenEnclaveJenkinsLibrary@${params.OECI_LIB_VERSION}"
+library "OpenEnclaveJenkinsLibraryCyan@${params.OECI_LIB_VERSION}"
```

Comment:
To be changed back to original before merging

Reply:
Due to the new additions to OpenEnclaveJenkinsLibrary in this PR, bors will fail if this line is reverted. I will make an update to this.